src/main/java/com/minecolonies/core/entity/ai/workers/crafting/EntityAIWorkAlchemist.java

```diff
@@ -24,7 +24,7 @@
 import com.minecolonies.core.colony.buildings.workerbuildings.BuildingAlchemist;
 import com.minecolonies.core.colony.interactionhandling.StandardInteraction;
 import com.minecolonies.core.colony.jobs.JobAlchemist;
-import com.minecolonies.core.entity.ai.basic.AbstractEntityAICrafting;
+import com.minecolonies.core.entity.ai.workers.crafting.AbstractEntityAICrafting;
```

Comment:
This import is technically not needed anymore (as the class is now in the same package). Same for most of the other AI classes in this package